.travis.yml

```diff
@@ -8,18 +8,18 @@ jobs:
   - stage: Code quality checks
     install: docker build --build-arg PYTHON_VERSION=3.6 --build-arg DJANGO_VERSION=1.11
       -t native -f Dockerfile-tmpl .
-    script: docker run -it --rm native sh -c 'pre-commit run --files $(git ls-files
+    script: docker run --rm native sh -c 'pre-commit run --files $(git ls-files
       transifex* tests*)'
   - stage: Test
     name: Test py2.7-dj1.11
     install: docker build --build-arg PYTHON_VERSION=2.7 --build-arg DJANGO_VERSION=1.11
       -t native -f Dockerfile-tmpl .
-    script: docker run -it --rm native pytest --cov --cov-report=term-missing
+    script: docker run -e CODECOV_TOKEN=$CODECOV_TOKEN --rm native sh -c 'pytest --cov --cov-report=term-missing && codecov'
```

Comment:
I'm a bit sceptical if we should include `xml` in `--cov-report` values (as described [here](https://docs.codecov.io/v4.3.0/docs/python). However it seems that the current script does the job in codecov.io.

Reply:
I think `codecov` doesn't require the report to be in XML. So I don't think we need it. If we were using another coverage tool we might.